#!/bin/sh -e

# DP: This patch was converted from pr26885-doc.dpatch
# DP: Original comment:
# DP:
# DP: Proposed patch for PR driver/26885 (doc files)

dir=
if [ $# -eq 3 -a "$2" = '-d' ]; then
    pdir="-d $3"
    dir="$3/"
elif [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch)
        patch $pdir -f --no-backup-if-mismatch -p0 < $0
        ;;
    -unpatch)
        patch $pdir -f --no-backup-if-mismatch -R -p0 < $0
        ;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1
esac
exit 0

--- doc/options.texi	2006-05-07 13:23:52.294265896 +0200
+++ doc/options.texi	2006-05-07 13:24:36.553537464 +0200
@@ -100,6 +100,12 @@
 ``f'', ``W'' or ``m'' are assumed to have a ``no-'' form unless this
 property is used.
 
+@item Negative(@var{othername})
+The option will turn off another option @var{othername}, which is the
+the option name with the leading ``-'' removed.  This chain action will
+propagate through the @code{Negative} property of the option to be
+turned off.
+
 @item Joined
 @itemx Separate
 The option takes a mandatory argument.  @code{Joined} indicates
